Packs/Arcanna/pack_metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "ArcannaAI",
+    "description": "Siscale Arcanna.Ai Cognitive automation platform that provides AI assistance to IT & Cybersecurity teams",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Siscale Engineering",
+    "created": "2021-06-16T20:46:43Z",
+    "url": "https://www.paloaltonetworks.com/cortex",
```

Comment:
Please put here valid URL users can visit if they have issues/questions about this pack.

Reply:
Done